### PR TITLE
MINOR: Bumped up PythonConnector MINOR version from 3.8.1 to 3.9.0

### PR DESCRIPTION
**Pull request by sfc-gh-yuwang**
@noreview - This is an automated process. No review is required

Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 - v3.9.0(TBD)
```
Comment (zzaym) [inline DESCRIPTION.md:11]: Remove this?

File: DESCRIPTION.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
 
+- v3.9.0(April 20,2024)
+
+   -N/A
```
Comment (zzaym) [inline DESCRIPTION.md:15]: Remove?

**Review by zzaym — APPROVED** [review]

File: DESCRIPTION.md (hunk 1 above)
Comment (sfc-gh-jdu) [inline DESCRIPTION.md:11]: Remove?

File: DESCRIPTION.md (hunk 2 above)
Comment (sfc-gh-jdu) [inline DESCRIPTION.md:15]: Remove?

**Review by sfc-gh-jdu — APPROVED** [review]

File: DESCRIPTION.md (hunk 2 above)
Comment (sfc-gh-mheavin) [inline DESCRIPTION.md]: What is N/A for?

File: DESCRIPTION.md [hunk 3]
```diff
@@ -10,6 +10,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
+
+
   - Added easy logging configuration so that users can easily generate log file by setup log config in `$SNOWFLAKE_HOME/config.toml`.
```
Comment (sfc-gh-mheavin) [inline DESCRIPTION.md:13]: For other drivers, we used "Added support for log settings in a logging configuration file."

**Pull request merged**